### ISSUE TITLE
Sizes not detected for accelerated networking

### ISSUE THREAD

**Issue by BernieWhite** [microsoft/PSRule.Rules.Azure#314]
**Description of the issue**

The size `Standard_E4s_v3` is not detected as supporting accelerated networking and should be.

Improve filter expression to better match VM sizes that support accelerated networking.

**Module in use and version:**

- Module: PSRule.Rules.Azure
- Version: **0.10.0-B2002023**

Captured output from `$PSVersionTable`:

```text
Name                           Value
----                           -----
PSVersion                      6.2.4
PSEdition                      Core
GitCommitId                    6.2.4
OS                             Microsoft Windows 10.0.18363
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```
